consensus/consensus.md

```diff
@@ -274,14 +274,31 @@ both the `Fee` and the `GasPrice`.
  -------------- -----
 | recipient    | 32  |
  -------------- -----
-| amount       | 8   |
+| amount       | var |
```

Comment:
`var` should be defined somewhere.

Reply:
Hmm, I just saw it being used further up in the same document so I assumed it to be defined...

Reply:
That is why I approved nonetheless.

Reply:
😅